knotx-core/knotx-fragment-assembler/src/main/java/com/cognifide/knotx/knot/assembler/FragmentAssemblerConfiguration.java

```diff
@@ -15,29 +15,26 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.cognifide.knotx.knot.templating.wrappers;
-
-import com.cognifide.knotx.fragments.Fragment;
+package com.cognifide.knotx.knot.assembler;
 
 import io.vertx.core.json.JsonObject;
 
-public class RawHtmlFragment implements HtmlFragment {
+class FragmentAssemblerConfiguration {
 
+  private final String address;
 
-  private final Fragment fragment;
+  private final AssemblyStrategy assemblyStrategy;
 
-  public RawHtmlFragment(Fragment fragment) {
-    this.fragment = fragment;
+  FragmentAssemblerConfiguration(JsonObject config) {
+    address = config.getString("address");
+    assemblyStrategy = AssemblyStrategy.valueOf(config.getString("assemblyStrategy"));
```

Comment:
To avoid problems, where user specify strategy in lowercase, do uppercase here to get the proper enum
```java
AssemblyStrategy.valueOf(config.getString("assemblyStrategy").toUpperCase()));
```